packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2604,20 +2599,6 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         const deserializedContent: ContainerRuntimeMessage = { type, contents };
         const serializedContent = JSON.stringify(deserializedContent);
 
-        if (serializedContent.length > defaultMaxOpSizeInBytes) {
```

Comment:
there's a test in `packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts` which would fail without this, so that should be adapted to the new error..

Reply:
I'll fix them as they show up in CI results :) First one are in.